helm/kubernetes-scanner/values.yaml

```diff
@@ -20,13 +20,21 @@
 
 enabled: true
 
-# TODO: set cluster name & orgID
-#organizationID: "setme"
+# MANDATORY: set cluster name & orgID
+#
+# organizationID: "setme"
+
+# MANDATORY: clusterName is a friendly name for the cluster that this agent is
+# running on. For example "prod-us" or "dev-eu".
+# If installing scanners to multiple clusters, please configure a different
+# cluster name for each cluster. These do not need to be globally unique, as
+# they are scoped by organization, but using clashing cluster names within an
+# organization may make it difficult for users to associate issues with specific
+# clusters later.
+#
+# clusterName: "default"
```

Comment:
Moved next to orgID and made mandatory, to avoid a footgun in which users can't distinguish data between clusters. If we merge this, we'll need to update our dogfood deployments' helm values.